Doc/library/xmlrpc.client.rst

```diff
@@ -548,6 +548,11 @@ Convenience Functions
    .. versionchanged:: 3.3
       The *use_builtin_types* flag was added.
 
+   .. versionchanged:: 3.12
```

Comment:
I'm not sure about this. There are lots of things in the stdlib that might internally convert an int to a str, and I don't think we've documented all of those individual things as separately "changed" -- at least grepping the docs for `versionchanged:: 3.12` I'm not finding such notes elsewhere. E.g. if we were generally doing this I'd also expect to find it in the JSON docs.

Reply:
Oh, I guess it is there in the JSON docs, it was just changed to a `versionchanged:: 3.11` because the security fix went into 3.11 too. See 34de67c094776bdc719d87f8810e053a2acc915a.

So I guess it makes sense to add it here too, but it should probably also say `versionchanged:: 3.11` here.

Reply:
It actually is in the json docs - which is really the only reason I added it here. The original PR added a `versionchanged:: 3.12` to the `.. function::load()` entry. It also added tests specifically in `test_json/test_decode.py` and `test_xmlrpc.py` so I found the lack of documentation related to XML odd.
That said, you are right that there are countless places where an int might get converted to a string or a string to an int, and not all are covered by documentation, or even a new test. I would be okay to drop this entry.

Reply:
Got your response after posting mine. New commit is up.

Reply:
I left if out of the xml docs on purpose as it'd just become a repeated message all over the place. JSON was one of the more common things parsing ints so that one stayed.